node.js/tests/utils.js

```diff
@@ -0,0 +1,24 @@
+// Set Ingestion Key:
+exports.setEnv = (key, value) => {
+    process.env[key] = value;
```

Comment:
In my opinion, this isn't a particularly helpful function, you could just inline this everywhere. It's probably even more clear that way.

Reply:
Done!